### PR TITLE
GameStop

### PR DESCRIPTION
**Pull request by aahurtado**

### CODE REVIEW

File: sites/gamestop.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        self.submit_order()
+
+
+
+    def create_msg(self, msg, status):
+        return {"msg": msg, "status": status}
+
+
+    # https://stackoverflow.com/questions/33225947/can-a-website-detect-when-you-are-using-selenium-with-chromedriver
+    def change_driver(self, loc):
```
Comment (Strip3s) [inline sites/gamestop.py:39]: We need to eventually move this into the util or a different shared class to emulate the browser more effectively for all sites.

File: sites/gamestop.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            fin.write(data)
+            fin.close()
+        else:
+            fin.close()
+
+
+    def init_driver(self):
+        driver_manager = ChromeDriverManager()
+        driver_manager.install()
+        # self.change_driver(driver_path)
```
Comment (Strip3s) [inline sites/gamestop.py:62]: Is this still needed?

File: sites/gamestop.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                  })
+                """
+        })
+
+        return browser
+
+
+    def login(self):
+        my_username = "username"
+        my_password = "password"
```
Comment (Strip3s) [inline sites/gamestop.py:78]: We need to add this to the settings page for login credentials per task.

**Review by Strip3s — CHANGES_REQUESTED** [review]
Need to make the login credentials configurable per task for more ease of use for users without the ability to address this within src.

File: sites/gamestop.py (hunk 1 above)
Comment (aahurtado) [reply sites/gamestop.py:39]: Agreed, all the driver code is left over from target.py.

File: sites/gamestop.py (hunk 3 above)
Comment (aahurtado) [reply sites/gamestop.py:78]: Agreed, needs to be added to GUI.

**Comment by aahurtado** [comment]
Added GameStop username and password to Settings GUI page. Gamestop password is also encrypted

PB

**Review by Strip3s — APPROVED** [review]

**Review by Strip3s — DISMISSED** [review]

**Review by Strip3s — APPROVED** [review]

**Comment by Strip3s** [comment]
ggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggfgggggggggggggggggggggggggggggggggggggagggggggggggggg gggggggggggggg

**Review by gr33k — APPROVED** [review]

**Pull request merged**